iommi/templates/iommi/form/heading.html

```diff
@@ -1,3 +1 @@
-<tr>
-    <th colspan="2">{{ field.display_name }}</th>
-</tr>
\ No newline at end of file
+{{ field.iommi_open_tag }}{{ field.display_name }}{{ field.iommi_close_tag }}
```

Comment:
Should the 'tr' be the default value for the tag (on the Field declaration or the base style)? Or is it that we haven't used the heading shortcut and is is not needed?

Reply:
I believe the tr/td thing is just left over from very old trioptima form code. So yea, we haven't used this in a while and that's why we didn't discover that it was broken.